scripts/skills/actives/rf_gouge_skill.nut

```diff
@@ -50,12 +50,17 @@ this.rf_gouge_skill <- ::inherit("scripts/skills/skill", {
 	function getTooltip()
 	{
 		local ret = this.getDefaultTooltip();
-		ret.push({
-			id = 6,
-			type = "text",
-			icon = "ui/icons/hitchance.png",
-			text = ::Reforged.Mod.Tooltips.parseString("Has a " + ::MSU.Text.colorizeMult(this.getInjuryThresholdMult()) + " lower [threshold|Concept.InjuryThreshold] to inflict [injuries|Concept.InjuryTemporary]")
-		});
+
+		local injuryThresholdMult = this.getInjuryThresholdMult();
+		if (injuryThresholdMult != 1.0)
+		{
+			ret.push({
+				id = 6,
```

Comment:
I would probably have started this at 10 just as any normal effect line on a skill.
It doesnt matter to me whether you got with 6 or 10 for this PR to be merged